src/translations/nbTranslations.json

```diff
@@ -210,10 +210,6 @@
     "yes": "Ja"
   },
   "disciplines": {
-    "0001": "Humaniora",
-    "0002": "Samfunnsvitenskap",
-    "0003": "Medisin og helsefag",
-    "0004": "Realfag og teknologi",
```

Comment:
disse skulle ikke blitt endret... i18n ally som rotert til :/

Reply:
Det skjer når man bruker `i18n ally` til å legge inn en ny key, som er mye enklere enn å legge til nøkkelen i json-fila selv. Det er jo slik vi helst vil gjøre det, men `i18n ally` endrer da sorteringen for disse. Mens en PR fra Lokalise vil endre tilbake igjen, slik at det blir kastet frem og tilbake. I praksis vil det virke som det skal da, så er litt fristet til å bare la det vær sånn faktisk. Mener det vil gi best developer experience